airflow/container.go

```diff
@@ -205,3 +205,39 @@ func getFmtEnvFile(envFile string, containerEngine Container) (string, error) {
 		return "", nil
 	}
 }
+
+func GetWebserverServiceName() string {
+	containerEngine := config.CFG.ContainerEngine.GetString()
+	switch containerEngine {
+	case string(DockerEngine):
+		return webserverServiceName
+	case string(PodmanEngine):
+		return config.CFG.WebserverContainerName.GetString()
+	default:
+		return webserverServiceName
+	}
+}
+
+func GetSchedulerServiceName() string {
+	containerEngine := config.CFG.ContainerEngine.GetString()
+	switch containerEngine {
+	case string(DockerEngine):
+		return schedulerServiceName
+	case string(PodmanEngine):
+		return config.CFG.SchedulerContainerName.GetString()
+	default:
+		return schedulerServiceName
+	}
+}
+
+func GetTriggererServiceName() string {
+	containerEngine := config.CFG.ContainerEngine.GetString()
+	switch containerEngine {
+	case string(DockerEngine):
+		return triggererServiceName
+	case string(PodmanEngine):
+		return config.CFG.TriggererContainerName.GetString()
```

Comment:
Why is it different in Podman ?

If we run multiple projects locally with Podman, we do not encounter the same "container name already exists" ?

Reply:
With Podman the entire logic is based on the container names, which could be configured by the user, whereas for Docker the logic is based on the service name, which is a static value in the docker-compose file.